src/GHCMod.hs

```diff
@@ -333,7 +342,14 @@ main = handler $ do
 
 progMain :: (Options,[String]) -> IO ()
 progMain (globalOptions,cmdArgs) = do
-    (res,_) <- runGhcModT globalOptions $ ghcCommands cmdArgs
+    let
```

Comment:
Here you go, no changes to globalOptions. I'd prefer to be able to ran most of this code while parsing options, but...